R/create-docs.sh

```diff
@@ -43,4 +45,7 @@ Rscript -e 'libDir <- "../../lib"; library(SparkR, lib.loc=libDir); library(knit
 
 popd
 
+# render creates SparkR vignettes
+Rscript -e 'library(rmarkdown); Sys.setenv(SPARK_HOME=tools::file_path_as_absolute("..")); render("pkg/vignettes/sparkr-vignettes.Rmd")'
```

Comment:
Similar to the previous line, lets add `libDir <- "../../lib"; library(SparkR, lib.loc=libDir);` here. I think that should make the Jenkins build work.